bin/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+ENV_TYPE=$1
+
+
+if test "$ENV_TYPE" = "production"
+then
+  SERVER_URL='trase.earth'
+else
+  SERVER_URL='staging.trase.earth'
+fi
+
+echo "DEPLOYING:  $ENV_TYPE ($SERVER_URL)"
+
+read -p "Press enter to continue"
+
+cp .env .env.tmp
+cp .env.$ENV_TYPE .env
+
+npm run webpack:prod
+
+cp .env.tmp .env
+rm .env.tmp
+
+scp -pr ./dist/* ubuntu@$SERVER_URL:/var/www/html/
```

Comment:
you should restart nginx varnish and apache after this. see the command I left you on slack

Reply:
does this clean /dist before it starts?

Reply:
> you should restart nginx varnish and apache after this. see the command I left you on slack

Even if we are just touching static files?